CHANGES.md

```diff
@@ -91,6 +91,8 @@ To be released.
     [[#228], [#234]]
  -  `Swarm.StartAsync()` now does not call `Swarm.StopAsync()` anymore,
     therefore `Swarm.StopAsync()` should be explicitly called.  [[#236]]
+ -  `Block<T>` became to store only one transaction per signer.  
```

Comment:
Since `Block<T>` does nothing with persist storage, we'd better use an other verb instead of “store” here, e.g.:

> A signer became to allowed to have only one transaction at most in a `Block<T>`.